zebra-consensus/src/transaction/tests.rs

```diff
@@ -177,6 +180,38 @@ fn v5_transaction_with_no_inputs_fails_validation() {
     );
 }
 
+#[tokio::test]
+async fn mempool_request_with_missing_input_is_rejected() {
```

Comment:
Can we have a test for the success case as well?

Reply:
Added a test for the success case in [`ad8d8ea`](https://github.com/ZcashFoundation/zebra/pull/5616/commits/ad8d8ea926c1415946f49b316c3c21616df6bb66).